src/index.js

```diff
@@ -1,14 +1,61 @@
-import _ from 'lodash';
 import './style.css';
+/* global loadTasksFromLocalStorage */
 
-function component() {
-  const element = document.createElement('div');
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Great job with your project, It would be really awesome if you could avoid disabling linters in your files. If it's not too much trouble, could you please remove this and the other eslint disable lines?